packages/web3-providers-ws/package.json

```diff
@@ -34,16 +36,19 @@
 		"eslint-config-prettier": "^8.3.0",
 		"eslint-config-web3-base": "0.1.0",
 		"eslint-plugin-import": "^2.25.3",
+		"express": "^4.18.1",
+		"http-proxy-middleware": "^2.0.6",
 		"jest": "^27.3.1",
 		"jest-extended": "^1.1.0",
+		"morgan": "^1.10.0",
```

Comment:
What is thing being used for? Doesn't seem to be used directly in these changes

Reply:
removed/ not necessary